docs/src/technik/coding_conventions/tests_naming.md

```diff
@@ -0,0 +1,62 @@
+# Naming Convention für Tests
+
+## Kontext
+
+Damit der Code übersichtlicher und einheitlicher ist, sollen Naming Conventions eingesetzt werden. Damit wird für Tests gewährleistet, dass deren Inhalt klar ist, ohne den Code lesen zu müssen.
```

Comment:
Hier gerne die aktuelle Situation konkreter Beschreiben damit der Hintergrund verständlich ist.

Reply:
siehe commit [b965b0a](https://github.com/it-at-m/Wahllokalsystem/pull/427/commits/b965b0a465a5c8c24d1a611666ee96bc340a6167)